EU4toV2/Source/V2World/Pops/PopType.h

```diff
@@ -0,0 +1,22 @@
+#ifndef POP_TYPE_H
+#define POP_TYPE_H
+
+#include "newParser.h"
+#include "Pop.h"
+#include <map>
+
+namespace mappers
+{
+	class PopType : commonItems::parser
```

Comment:
Since it holds a bunch of types, it might be better to pluralize the name.

Reply:
done.